src/BannerContent.tsx

```diff
@@ -13,7 +13,8 @@ export type Props = {
   link?: {
     msg?: string,
     url: string,
-    target?: '_blank' | '_self' | '_parent' | '_top' | 'framename'
+    target?: '_blank' | '_self' | '_parent' | '_top' | 'framename',
+    rel: string,
```

Comment:
this should be optional, you can mark it as optional by adding the `?` before the colon

Reply:
@FrancescoCioria good catch, thanks! I've just fixed it.